ftp/tftp/src/opnsense/mvc/app/views/OPNsense/Tftp/general.volt

```diff
@@ -26,6 +26,15 @@
  #}
 
 <div class="content-box" style="padding-bottom: 1.5em;">
+    <div id="about">
+        <table class="table-responsive table table-striped">
+            <tr>
+                <td>
+                The root folder for transfering files is /usr/local/tftp.
```

Comment:
I would make that translatable, but otherwise the string is ok.

Did not code in volt since a long time but afaik it was {{ _.lang('xxx') }}.

Reply:
sorry, it is the opposite direction: lang._

Reply:
Correct, fixed! Thx for your review :)